recipes/openfast/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "openfast" %}
+{% set version = "v2.3.0" %}
```

Versions shouldn't have a v in them:
```suggestion
{% set version = "2.3.0" %}
```